tensorflow_probability/python/internal/hypothesis_testlib.py

```diff
@@ -77,7 +77,7 @@ def tfp_hp_settings(default_max_examples=None, **kwargs):
       deadline=None,
       suppress_health_check=[hp.HealthCheck.too_slow],
       max_examples=hypothesis_max_examples(default=default_max_examples),
-      print_blob=hp.PrintSettings.ALWAYS)
```

Comment:
This is a nice catch, but actually, we are (for the moment) stuck with an even older version of Hypothesis internally.  Therefore, please revert this line :(

I'm making an internal change to our Travis config to pin the Hypothesis version to a known-good one.

Reply:
Cool, thanks, reverted in https://github.com/tensorflow/probability/pull/753/commits/6e191876fc019108d60d9c5286b17f4f569213db